config/csm/csm_config.yaml

```diff
@@ -84,11 +84,11 @@ Restcall_LC:
   s3accounts_endpoint : "/api/v2/s3_accounts"
   s3_bucket_endpoint : "/api/v2/s3/bucket"
   IAM_users_endpoint : "/api/v2/iam_users"
-  s3_iam_user_endpoint : "/api/v2/s3/iam/users"
+  s3_iam_user_endpoint : "/api/v2/iam/users"
   s3_iam_keys_endpoint : "/api/v2/s3/iam/keys"
-  s3_iam_caps_endpoint : "/api/v2/s3/iam/caps"
+  s3_iam_caps_endpoint : "/api/v2/iam/caps"
   get_set_quota: "/api/v2/iam/quota/{}"
-  csmuser_endpoint : "/api/v2/csm/users"
+  csmuser_endpoint : "/api/v2/system/users"
   csmuser_permission_endpoint: "/api/v2/permissions"
   bucket_policy_endpoint : "/api/v2/s3/bucket_policy/{}"
   perf_stats_endpoint: "/api/v2/metrics/perf"
```

Comment:
degraded_cap_endpoint: "/api/v2/capacity/status" this needs to be changed to degraded_cap_endpoint: "/api/v2/capacity/status"